tasks/release.rb

```diff
@@ -44,13 +44,21 @@
       raise "Could not insert PRE in #{file}" unless $1
 
       File.open(file, 'w') { |f| f.write ruby }
+
+      if framework == "actioncable"
```

Comment:
Can check for `if File.exist?("#{framework}/package.json")` instead of special-casing AC.